int32 player_x
int32 player_y
int32 score
string state
int64 lives